src/sonic_ax_impl/mibs/ietf/rfc4292.py

```diff
@@ -35,7 +35,12 @@ def update_data(self):
                 ipn = ipaddress.ip_network(ipnstr)
                 ent = self.db_conn.get_all(mibs.APPL_DB, routestr, blocking=True)
                 nexthops = ent[b"nexthop"].decode()
-                for nh in nexthops.split(','):
+                ifnames = ent[b"ifname"].decode()
+                for nh, ifn in zip(nexthops.split(','), ifnames.split(',')):
```

Comment:
The code here tolerates this case. zip() just returns a list as long as the shorter one of nexthops and ifnames.